Cargo.toml

```diff
@@ -31,6 +31,6 @@ repository = "https://github.com/sunchao/parquet-format-rs"
 keywords = ["parquet", "hadoop"]
 
 [dependencies]
-ordered-float = "1.0.1"
-thrift = "0.0.4"
-try_from = "0.2.2"
\ No newline at end of file
+ordered-float = "1.0.2"
```

Comment:
Can we use semantic versions like `ordered-float = 1.0`? we can automatically get updates from minor version changes.

Reply:
Yes sure.